### PR TITLE
b2c_navigation_endpoints

### PR DESCRIPTION
**Pull request by shubham-khandelwal2-metacube**
## PR Description

b2c_navigation_endpoints
https://spryker.atlassian.net/browse/CC-16326

### CODE REVIEW

**Review by OKrivtsova — COMMENTED** [review]
please recreate PR correctly and fix folder structure, then I will be able to actually check tests

File: tests/api/b2c/glue/navigation_endpoints/navigations/negative.robot [hunk 1]
```diff
@@ -0,0 +1,19 @@
+*** Settings ***
+Suite Setup    SuiteSetup
+Test Setup    TestSetup
+Resource    ../../../../../resources/common/common_api.robot
```
Comment (OKrivtsova) [inline tests/api/b2c/glue/navigation_endpoints/navigations/negative.robot:4]: after changing folder structure this is no longer a valid address, please fix

File: tests/api/b2c/glue/navigation_endpoints/navigations/positive.robot [hunk 2]
```diff
@@ -0,0 +1,71 @@
+*** Settings ***
+Resource    ../../../../../resources/common/common_api.robot
```
Comment (OKrivtsova) [inline tests/api/b2c/glue/navigation_endpoints/navigations/positive.robot:2]: after changing folder structure this is no longer a valid address, please fix

Comment (shubham-khandelwal2-metacube) [reply tests/api/b2c/glue/navigation_endpoints/navigations/positive.robot]: DONE

File: tests/api/b2c/glue/navigation_endpoints/navigations/negative.robot (hunk 1 above)
Comment (shubham-khandelwal2-metacube) [reply tests/api/b2c/glue/navigation_endpoints/navigations/negative.robot]: DONE

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**